emcc.py

```diff
@@ -62,8 +62,8 @@
 
 LIB_PREFIXES = ('', 'lib')
 
-JS_CONTAINING_SUFFIXES = ('js', 'html')
-EXECUTABLE_SUFFIXES = JS_CONTAINING_SUFFIXES + ('wasm',)
+JS_CONTAINING_SUFFIXES = ('.js', '.html')
```

Comment:
we sort of had an informal distinction between "endings" (see before this part of the diff) which contain the dot, and "suffixes" which do not. this PR makes them both include the dot, so perhaps we should rename them all to either suffixes or endings?